subsys/net/ip/Kconfig

```diff
@@ -754,6 +754,7 @@ config NET_BUF_DATA_POOL_SIZE
 
 config NET_PKT_BUF_USER_DATA_SIZE
 	int "Size of user_data available in rx and tx network buffers"
+	default BT_L2CAP_TX_USER_DATA_SIZE if NET_L2_BT
 	default 4
 	range 4 16
```

Comment:
Given the build asserts, could we modify the range instead? i.e.
```
range BT_L2CAP_TX_USER_DATA_SIZE 16 if NET_L2_BT
```

That way we should be able to get rid of the 
```c
BUILD_ASSERT(CONFIG_NET_PKT_BUF_USER_DATA_SIZE >= CONFIG_BT_L2CAP_TX_USER_DATA_SIZE,
	     "net buf user data size is too small for sending over L2CAP.");
```

at least, right?

Reply:
I think that could work, yes

Reply:
alright, tried it, I still have to have the default or it's bad UX, the build just fails if the user doesn't specify the size manually. Adding the range to get rid of the assert is still valid though